tests/Support/Builders/UserBuilder.php

```diff
@@ -29,10 +29,10 @@ public function associations(): array
     public function filters(): array
     {
         return [
-            'name'       => $this->filter()->expectMany('string')->byField('name'),
-            'created_at' => $this->filter()->expect('datetime')->byField('created_at', '>'),
-            'posts'      => $this->filter()->expectMany('string')->byAssociation('posts', 'id'),
-            'updated_at' => $this->filter()->expect('date', 'd-m-Y')->byField('updated_at', '>'),
+            'name'       => $this->filter()->expect()->string()->byField('name'),
+            'created_at' => $this->filter()->expect()->datetime()->byField('created_at', '>'),
+            'posts'      => $this->filter()->expect()->string()->byAssociation('posts', 'id'),
```

Comment:
See my comments above from the `ExceptionHandlingTest`.

Reply:
Not sure what you mean here

Reply:
To fix the the `it_can_filter_on_associations` test, the filter definition should be changed.

Reply:
Ahh like that, adjusted it, thought this was about the date filter somehow due to that being the only change in this file before adjusting the filter so the test completes again